From: Boyuan Yang <byang@debian.org>
Date: Sat, 1 Jun 2024 22:14:56 -0400
Subject: tests/compat/py39.py: Drop py39compat to avoid jaraco.test

We directly use import_helper and os_helper from python test.support
library. This drops compatibility with Python 3.9 or earlier, but
avoids the usage of jaraco.test library which is not packaged yet.
---
 tests/compat/py39.py | 21 ++++++++++++++-------
 1 file changed, 14 insertions(+), 7 deletions(-)

diff --git a/tests/compat/py39.py b/tests/compat/py39.py
index 16c8b57..0287a12 100644
--- a/tests/compat/py39.py
+++ b/tests/compat/py39.py
@@ -1,9 +1,16 @@
-from jaraco.test.cpython import from_test_support, try_import
+# Debian-specific: Drop compatibility with python 3.10 or earlier
+# Do not use jaraco.test, not packaged
 
+#from jaraco.test.cpython import from_test_support, try_import
 
-os_helper = try_import('os_helper') or from_test_support(
-    'FS_NONASCII', 'skip_unless_symlink'
-)
-import_helper = try_import('import_helper') or from_test_support(
-    'modules_setup', 'modules_cleanup'
-)
+
+#os_helper = try_import('os_helper') or from_test_support(
+#    'FS_NONASCII', 'skip_unless_symlink'
+#)
+#import_helper = try_import('import_helper') or from_test_support(
+#    'modules_setup', 'modules_cleanup'
+#)
+
+# Debian-specific: Use Python standard library
+from test.support import os_helper
+from test.support import import_helper
